src/APIFunctions/S3Bucket.js

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
+
+export function uploadToS3(files) {
+  for (let i = 0; i < files.length; i++) {
+    let file = files[i];
+    let fileParts = files[i].name.split('.');
+    let fileName = fileParts[0];
+    let fileType = fileParts[1];
+    axios
+      .post('http://localhost:8082/api/S3Bucket/getSignedUrl', {
+        fileName: fileName,
+        fileType: fileType,
+      })
+      .then((response) => {
+        const returnData = response.data.url;
+        const signedRequest = returnData.signedRequest;
+
+        const options = {
+          headers: {
+            'Content-Type': fileType,
+          },
+        };
+        axios
+          .put(signedRequest, file, options)
```

Comment:
is this to download the file?

Reply:
This is to upload the file to S3. Does something look off?

Reply:
some comments explaining what's going on would be great